bigbluebutton-html5/imports/startup/client/routes.js

```diff
@@ -12,6 +12,12 @@ const browserHistory = useRouterHistory(createHistory)({
   basename: Meteor.settings.public.app.basename,
 });
 
+const disconnect = () => {
```

Comment:
Added a disconnect on ending/error routes just for the sake of sanity but the server should cut their connections anyway.